ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -756,6 +978,20 @@ literals:
 - position_status
 - condensing_water_valve_percentage_sensor
 - entering_cooling_coil_temperature_sensor
+- condensing_supply_water_isolation_valve_open_status
+- condensing_supply_water_isolation_valve_closed_status
+- condensing_supply_water_isolation_valve_open_command
+- condensing_supply_water_isolation_valve_closed_command
+- entering_chilled_water_temperature_sensor
+- leaving_chilled_water_temperature_sensor
+- line_voltage_sensor
+- inhibit_start_time_command
```

Comment:
what is meant by this? and why is it needed in this integration?

Reply:
It has been removed